City-of-Helsinki/drupal-helfi-etusivu
# UHF-5283 main news styles

# [UHF-5283](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5283)

Changes to main news view mode and new templates for main news

## How to install
Update first hdbt and platform config
* `composer require drupal/hdbt:dev-UHF-5283-main-news-styles`
* `composer require drupal/helfi_platform_config:dev-UHF-5283-main-news-styles`
* `drush updb; drush cr`
After updating hdbt and platform config
* Checkout branch and run `drush cim -y; drush cr`

## How to test

Go to https://helfi-etusivu.docker.so/fi/uutiset and check that the main news view is now using the card teaser view mode and is styled according to the design.

## Other PRs
* https://github.com/City-of-Helsinki/drupal-hdbt/pull/326
* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/322
